src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -82,13 +87,23 @@ export const MapRoom: React.FC<MapRoomProps> = ({
     <div
       className={containerClasses}
       style={roomInlineStyles}
-      onClick={isUnclickable ? noop : selectRoomWithSound}
-      onMouseEnter={isUnclickable ? noop : handleRoomHovered}
-      onMouseLeave={isUnclickable ? noop : handleRoomUnhovered}
+      onClick={isUnclickable || isIframe ? noop : selectRoomWithSound}
+      onMouseEnter={isUnclickable || isIframe ? noop : handleRoomHovered}
+      onMouseLeave={isUnclickable || isIframe ? noop : handleRoomUnhovered}
```

Comment:
Using the `isNonInteractableRoom` pattern mentioned in another comment in this review:

```suggestion
      onClick={isNonInteractableRoom ? noop : selectRoomWithSound}
      onMouseEnter={isNonInteractableRoom ? noop : handleRoomHovered}
      onMouseLeave={isNonInteractableRoom ? noop : handleRoomUnhovered}
```

Reply:
done